src/fmu/dataio/aggregation.py

```diff
@@ -234,16 +234,15 @@ def _generate_aggrd_metadata(
         # next, the new object will trigger update of: 'file', 'data' (some fields) and
         # 'tracklog'.
 
-        # Get content and make a temporary config from template to be allowed to
+        # Make a temporary config from template to be allowed to
         # initialize a temporary ExportData without warnings so that we can get to the
         # objectdata_provider
-        content = template["data"]["content"]
         config = {
             "access": template["access"],
             "masterdata": template["masterdata"],
             "model": template["fmu"]["model"],
         }
-        etemp = dataio.ExportData(config=config, name=self.name, content=content)
+        etemp = dataio.ExportData(config=config, name=self.name)
```

Comment:
no need to provide content here. We only initialize this ExportData object in order to get a objdata provider, and we only use  the objdata provider to get the `bbox` and `extension` and they are not dependent on the content.